js/course.js

```diff
@@ -92,6 +92,26 @@ function setCourseOptionsContent(modal, options) {
             }
         }
     }
+
+    // Add request icon button only if default icon doesn't already exist
+    let hasDefaultIcon = (function (course) {

```

Comment:
Also the name tripped me up at first: isn't the icon given by the `.` regex the default icon? Perhaps `hasCustomIcon` is more appropriate? I'm not sure.